cmd/cdi-cloner/clone-source.go

```diff
@@ -209,7 +209,7 @@ func getInputStream(preallocation bool) (rc io.ReadCloser) {
 	default:
 		klog.Fatalf("Invalid content-type %q", contentType)
 	}
-	return
+	return rc
```

Comment:
Since we are being explicit with the return I think it'd be better to drop the named return parameter in most of these functions.

Reply:
Ok sounds good. Many of these functions can use a refactor.